Assets/HoloToolkit/Input/Scripts/Focus/InputSourcePointer.cs

```diff
@@ -20,38 +20,50 @@ public class InputSourcePointer : IPointingSource
 
         public bool OwnAllInput { get; set; }
 
-        public Ray Ray
```

Comment:
Same here. Mark obsolete and just return the first ray `Rays[0]`

Reply:
Works for me.

Reply:
Please provide instructions here too.